src/TodoList.coffee

```diff
@@ -0,0 +1,53 @@
+define 'TodoList',
+[
+    './ViewInterface.coffee'
+],
+(ViewInterface) ->
+    class TodoList extends ViewInterface
+        items = ['First todo']
+        todoListSelector = document.querySelector('.todo-list')
+        init: ->
```

Comment:
empty line after variable declaration. 
And separate each method with empty line